MSAL/src/MSALErrorConverter.m

```diff
@@ -153,4 +153,14 @@ - (NSError *)errorWithDomain:(NSString *)domain
     return [NSError errorWithDomain:msalDomain code:msalErrorCode userInfo:msalUserInfo];
 }
 
+- (NSString *)oauthErrorKey
```

Comment:
Why is this necessary :)?

Reply:
Each error converter uses different keys for storing OAuth errors, so we didn't know which string key to use. For example in broker, if we include only CommonCore, OAuth errors would be stored under MSID keys, but if we also include MSAL, OAuth errors would be stored under MSAL keys.

See also a PR in [common core](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/292).